src/dataService/savedItemsService.ts

```diff
@@ -379,8 +379,11 @@ export class SavedItemDataService {
    * the list table, by item id.
    * @param itemIds The item IDS to update the `time_Updated` to now.
    */
-  public updateListItemMany(itemIds: string[]): Knex.QueryBuilder {
-    return this.listItemUpdateBuilder().whereIn('item_id', itemIds);
+  public updateListItemMany(
```

Comment:
Add an optional timestamp to these functions so they can be reused (without having to refactor any other code)